sdk/vaa/payloads.go

```diff
@@ -276,6 +276,9 @@ type (
 	}
 )
 
+// However, this would be a breaking change for integrators that use this function
```

Comment:
The "However" sounds odd here.

Reply:
Ah yeah thanks, the order of these lines is backward. I can fix this